pkgs/development/libraries/giflib/default.nix

```diff
@@ -40,7 +29,9 @@ stdenv.mkDerivation rec {
     ./mingw-install-exes.patch
   ];
 
-  nativeBuildInputs = lib.optionals stdenv.isDarwin [
+  nativeBuildInputs = [
+    imagemagick_light
```

Comment:
This is only used for HTML docs. Fixed in https://github.com/NixOS/nixpkgs/pull/302500.